README.md

```diff
@@ -209,26 +209,31 @@ node src/bin/facilitator.js progressRedeem path_to_config.json messageHash
 
 ```
 
- * `messageHash` generated in redeem facilitator step.
-
-## Deploy TokenRules
-
-Prerequisite: `eip20Token` and `organization` in your config file.
-
-```bash
-# Help:
-node ./src/bin/tokenrules.js --help
-
-# Deploy TokenRules token:
-node ./src/bin/tokenrules.js config.json eip20Token organization
-```
-
-It will write `tokenRulesAddress` to your config file.
-
-* Replace `config.json` with the path to the configuration file.
-* Replace `eip20Token` with eip20Token address.
-* Replace `organization` with an organization contract address. 
-
+  * `messageHash` generated in redeem facilitator step.
+  
+ ## Setup OpenST
+    
+ ```bash
+ # Help:
+ node ./src/bin/openst.js --help
+    
+ # Setup OpenST:
+ node ./src/bin/openst.js config.json organization eip20Token
+ ```
+    
+ It will write below contract address to your config file:
+ * TokenHolder master copy address.
+ * Gnosis master copy address.
+ * Recovery master copy address.
+ * UserWalletFactory contract address.
+ * ProxyFactory contract address.
+ * CreateAndAddModule helper contract address.
+ * TokenRules contract address.
+    
+ * Replace `config.json` with the path to the configuration file.
+ * Replace `organization` with an organization contract address. 
+ * Replace `eip20Token` with eip20Token address.
+ 
```

Comment:
Why did you add a leading space to every line?

Reply:
It's not needed. Removed it 👍